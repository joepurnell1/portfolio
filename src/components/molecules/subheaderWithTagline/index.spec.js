import React from 'react';
import renderer from 'react-test-renderer';
import SubHeaderWithTagline from './index';

describe('Subheader with Tagline component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <SubHeaderWithTagline
        headerText="The people that know me best"
        taglineText="The key that I won't forget, too soon"
        colour="#530"
      />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
