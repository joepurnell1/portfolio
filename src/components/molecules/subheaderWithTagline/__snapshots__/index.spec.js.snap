// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subheader with Tagline component should render default as expected 1`] = `
<div>
  <h2
    className="sc-bdVaJa iiEJiO"
  >
    The people that know me best
  </h2>
  <h4
    className="sc-htpNat byCFkh sc-bwzfXH fdnVJW"
  >
    The key that I won't forget, too soon
  </h4>
</div>
`;
